components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/AbstractApplicationAuthenticator.java

```diff
@@ -153,43 +155,77 @@ protected boolean isStepHasMultiOption(AuthenticationContext context) {
         return stepHasMultiOption;
     }
 
-    protected void publishAuthenticationStepAttempt(HttpServletRequest request, AuthenticationContext context,
-                                                  User user, boolean success) {
+    protected void publishAuthenticationStepAttemptSuccess(HttpServletRequest request, AuthenticationContext context,
```

Comment:
Changing the method signature will effect several places. Did you able to check those places?

Reply:
you can't change the method signature of protected/public methods.

Reply:
yes, I checked in all the places and the tests. Also discussed in the code review.

Reply:
But customers are using this AbstractApplicationAuthenticator to write their own custom authenticators, so changing the method signature is not acceptable. Is it decided to change the protected method signature in the code review?

Reply:
ACK and Changed